.github/workflows/ci.yml

```diff
@@ -1019,8 +1019,15 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - { sys: mingw64, env: x86_64 }
-          - { sys: mingw32, env: i686 }
+          - sys: mingw32
+            env: i686
+            install_mingw64_only: ""
+          - sys: mingw64
+            env: x86_64
+            install_mingw64_only: |
```

Comment:
I'd name this "extra_install", I think, since it's there to install extra stuff. And you can leave it off on the other job, variables default to empty.

Reply:
Done: commit bb8a7940aa0b418ee85061bb1eca37488932c597